src/components/QuranReader/ReadingView/Line.tsx

```diff
@@ -40,13 +50,20 @@ const Line = ({ lineKey, words, isBigTextLayout, pageIndex, lineIndex }: LinePro
   const firstWordData = getWordDataByLocation(words[0].location);
   const shouldShowChapterHeader = firstWordData[1] === '1' && firstWordData[2] === '1';
 
+  const { quranFont, quranTextFontScale } = useSelector(
+    selectQuranReaderStyles,
+    shallowEqual,
+  ) as QuranReaderStyles;
+
   return (
     <div
       ref={selectedItemRef}
       id={lineKey}
       className={classNames(styles.container, {
         [styles.highlighted]: isHighlighted,
         [styles.mobileInline]: isBigTextLayout,
+        [styles[getLineWidthClassName(quranFont, quranTextFontScale)]]: isFixedWidth,
```

Comment:
as mentioned above, this should always be applied so we can add it as a param inside classNames e.g.`classNames(styles.container, styles[getLineWidthClassName(quranFont, quranTextFontScale)]`